client/elements/addons/sc-text-functions.html

```diff
@@ -0,0 +1,96 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/app-localize-behavior/app-localize-behavior.html">
+<link rel="import" href="../../bower_components/iron-a11y-keys/iron-a11y-keys.html">
+<!--
+  Static page functions
+-->
+<dom-module id="sc-text-functions">
+  <template>
+  </template>
+
+  <script>
+        /**
+       * @extends {Polymer.Element}
+       * @appliesMixin Polymer.AppLocalizeBehavior
+       */
+      class SCTextFunctions extends Polymer.mixinBehaviors([Polymer.AppLocalizeBehavior], ReduxMixin(Polymer.Element)) {
+
+          static get properties() {
+              return {
+                  inputElement: {
+                      type: Object
+                  },
+                  language: {
+                      type: String,
+                      statePath: 'siteLanguage'
+                  },
+                  fallbackLanguage: {
+                      type: String,
+                      value: 'en'
+                  },
+                  showParagraphs: {
+                      type: Boolean,
+                  },
+                  chosenTextView: {
+                      type: String,
+                  },
+                  lang: {
+                      type: String
+                  }
+              }
+          }
+
+          connectedCallback() {
+              super.connectedCallback();
+          }
+
+          deathToTheBeast() {
+              const inputText = this.inputElement.innerHTML;
```

Comment:
it's better to pass `this.inputElement` as an argument instead of using `this` in the function body, this way the function doesn't touch variables outside of its scope.

Reply:
I tried that but if does not seem to like that. Maybe I just don't have my day today but the a11y element just wants to have a function without arguments and I do not see how else to pass that element to the function.